corehq/apps/userreports/tests/test_report_aggregation.py

```diff
@@ -533,7 +583,7 @@ def test_total_row_with_expanded_column(self):
                     ],
                     [3, 1, 0, 3],
                     [6, 0, 2, 2],
-                    [9, 1, 2, 5],
```

Comment:
I guess it depends on what you expect from a totals column. Not obvious to me that there is a single correct answer for a 'min' column.